homeassistant/components/nissan_leaf/sensor.py

```diff
@@ -75,6 +80,10 @@ def __init__(self, car, ac_on):
         """Set up range sensor. Store if AC on."""
         self._ac_on = ac_on
         super().__init__(car)
+        if ac_on:
```

Comment:
How would the unique ID change if the AC is on or not?

Reply:
Nissan APIs returns two figures for the range of the car, one for when the AC is on, and one for when the AC is off.  We create two sensors to expose the range when AC is on and AC is off, so need a unique ID.